tools/pubsys/src/aws/publish_ami/mod.rs

```diff
@@ -503,31 +494,30 @@ pub(crate) async fn modify_regional_images(
 
 mod error {
     use crate::aws::{self, ami};
-    use rusoto_core::RusotoError;
-    use rusoto_ec2::{ModifyImageAttributeError, ModifySnapshotAttributeError};
+    use aws_sdk_ec2::error::{
+        DescribeImagesError, ModifyImageAttributeError, ModifySnapshotAttributeError,
+    };
+    use aws_sdk_ec2::types::SdkError;
     use snafu::Snafu;
     use std::io;
     use std::path::PathBuf;
 
     #[derive(Debug, Snafu)]
     #[snafu(visibility(pub(super)))]
     pub(crate) enum Error {
-        #[snafu(display("Error creating {} client in {}: {}", client_type, region, source))]
-        Client {
-            client_type: String,
+        #[snafu(display("Error creating client config in {}: {}", region, source))]
+        ClientConfig {
```

Comment:
We removed the `client_type` field from this error. When something does fail with the client config generation, is it clear which type of client (e.g. EC2 client) we're generating the config for? Or does it not matter?